lib/table_rex/renderer/text.ex

```diff
@@ -96,9 +96,10 @@ defmodule TableRex.Renderer.Text do
 
   defp render_title({%Table{title: title} = table, meta, opts, rendered}) do
     inner_width = Meta.inner_width(meta)
-    line = do_render_cell(title, inner_width)
-    if meta.render_vertical_frame? do
-      line = line |> frame_with(opts[:vertical_symbol])
+    line = if meta.render_vertical_frame? do
+      do_render_cell(title, inner_width) |> frame_with(opts[:vertical_symbol])
+    else
+      do_render_cell(title, inner_width)
     end
```

Comment:
I think to reduce duplication and increase readability here it'd be nice to do the actions which are common to both branches _before_ the branch. Like so:

```elixir
line = do_render_cell(title, inner_width)
line = if meta.render_vertical_frame? do
  line |> frame_with(opts[:vertical_symbol])
else
  line
end
```
In this case it's really simple to see with a glance that only the successful branch "does something extra".